doc/howto/dev/contribute_to_paddle_cn.md

```diff
@@ -1,130 +1,185 @@
 # 如何贡献代码
 
 我们真诚地感谢您的贡献，欢迎通过 GitHub 的 fork 和 pull request 流程来提交代码。
- 
+
 ## 代码要求
 - 你的代码必须完全遵守 [doxygen](http://www.stack.nl/~dimitri/doxygen/) 的样式。
 - 确保编译器选项 WITH\_STYLE\_CHECK 已打开，并且编译能通过代码样式检查。
 - 所有代码必须具有单元测试。
```

Comment:
I created an issue https://github.com/PaddlePaddle/Paddle/issues/1916 about this.